source/usage-examples/updateMany.txt

```diff
@@ -0,0 +1,113 @@
+===========================
+Update a Multiple Documents
+===========================
+
+.. default-domain:: mongodb
+
+You can update multiple documents using the ``updateMany()`` method on
+a ``MongoCollection`` object. The method accepts a **filter** that matches the
+document you want to update and an **update** statement that instructs the
+driver to change the matching document. The ``updateMany()`` method only
+updates all the documents in the collection that match the filter.
+
+To perform an update with the ``updateMany()`` method, you must pass
+a query filter and an update document. The query filter specifies the criteria
+for which document to perform the update on and the update document provides
+instructions on what changes to make to them.
```

Comment:
This sentence is a little bit of a mouthful for me, would it make sense to break this up into two sentences, each explaining a single parameter? "which document to perform the update on" also implies that this method only updates a single document -- can we pluralize that?

Reply:
I think I accidentally left "document" in there, so it doesn't belong there. I'll update to condense the sentence.